text/0000-project-descriptor.md

```diff
@@ -0,0 +1,150 @@
+# Meta
+[meta]: #meta
+- Name: Minimal Project Descriptor
+- Start Date: 2019-06-11
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: https://github.com/buildpack/rfcs/pull/25
+
+# Summary
+[summary]: #summary
+
+This is a proposal for a new file, `project.toml`, containing configuration for apps, services, functions, and buildpacks.
```

Comment:
While `.toml` is a great readable format, I have concerns that picking `.toml` might add more friction to users coming from the container world, which seems to have converged towards using `.yaml`. In my opinion, this is a data point that should be factored in when picking a format.

As a middle ground, maybe this proposal could say that `.yaml` and `.json` format are also accepted, but `.toml` is preferred (in the end, they all represent structured data, and `.yaml` is even a superset of `.json`).

Reply:
I think we should stick to `toml`. My reasoning is that all other parts of the buildpack spec use yaml, and as a platform maintainer, it would be quite frustrating to have to support other formats both in the platform itself and in support requests coming in from users.

The name `project.toml` also has fewer (no?) conflicts with other projects (I see that references to `project.yaml` in OpenStack and ArgoCD).

The `json` format doesn't allow for comments, which is why it wasn't picked.